payroll/models.py

```diff
@@ -241,6 +242,32 @@ def clean(self):
             raise ValidationError('{} name must be unique.'.format(self))
 
 
+class EmployerHighestSalaries(pg.MaterializedView):
+    '''
+    Materialized view of employer total pay, for retrieving highest salaries
+    in a given year.
+    '''
+    sql = '''
+        SELECT
+          salary."id" as payroll_salary_id,
+          employer.id as employer_id,
```

Comment:
could we combine this as 

```sql
coalesce(empolyer.id, employer.parent_id as target_employer_id
```

and avoid the OR conditions below?

Reply:
that will always return employer id -- the other way, it wouldn't return the right thing for departments :-(

Reply:
what if parent_id was first, i meant.